harmony/src/main/java/com/frybits/harmony/internal/HarmonyJsonUtils.kt

```diff
@@ -46,141 +61,123 @@ private const val BOOLEAN = "boolean"
 private const val STRING = "string"
 private const val SET = "set"
 
+private val json = Json {
+    prettyPrint = false
+    isLenient = false
+}
+
+@JvmSynthetic
+@Throws(IOException::class, SerializationException::class)
+internal fun <T : Reader> T.readHarmony(): Pair<String?, Map<String?, Any?>> {
```

Comment:
Crazy thought/question - Could we configure from the client side which JSON serializer to use?  Thinking about it like this:

1. We take in the new version of harmony 
2. Configure the Harmony SDK to have x% of users leveraging the new json serializer and rest still chugging along on the old serializer

I think that makes it easier for us to safely adopt and migrate and we could report issues that we see as we're onboarding.  We could also just force the old serializer if we hit something big enough that causes trouble.